server/models/db/school.js

```diff
@@ -0,0 +1,58 @@
+const _ = require('lodash');
+
+const { db } = require('../../lib/connection');
+
+const UserTypes = require('../../constants/userTypes.json');
+const Tables = require('../../constants/tables.json');
+
+const WorkerDetails = require('../data/Worker');
+
+// This method returns the corresponding workers given school ID. If the school ID is incorrect, it will return null. 
+// If either the tableName or Object are null or undefined, an error will be thrown.
+async function getWorkerIdsForSchool(schoolId) {
+    if (_.isNil(schoolId)) return null;
+    try {
+        //Query to recieve worker IDs from Service Worker School Table 
+        const workerIds = await db.any(`select worker_id from Service_Worker_School where school_id='${schoolId}';`);
+        //return workerIds;
+
+        var queryParams = `${Tables.worker}.worker_id, ${Tables.worker}.first_name, ${Tables.worker}.last_name`;
+        var tableJoins = ` ${Tables.worker} `;
+        //queryCond += ` and ${Tables.appointment}.student_id=${studentId}`;
+        //workerIds = 8000000;
+        const workerIds_2 = JSON.stringify(workerIds);
+
+        //up till here, this is relevant to getting the worker ids 
+
+        //const query_string = workerIds_2.data.join();
+        /*
+        //Parse input JSON string 
+        const object = JSON.parse(workerIds_2);
+        //Join elements of data array in object to a comma separated string
+        const query_string = object.data.join();
+        
+        query_string = '';
+        for (x in workerIds_2){
+            query_string += workerIds_2;
+        } 
+        */
+
+
+        // this part is relevant to getting the worker objects from the ids 
+        const queryStmt = `select ${queryParams} from ${tableJoins} where worker_id in (${query_string});`;
```

Comment:
Hi Simran,
As previously mentioned, I would recommend moving this logic to its own method as it can be reused for other functionalities as well. The new method would take in the array of workerIds and return all the worker details associated with these workerIds.

Reply:
Hi Amy! 

Thank you this has been resolved in the latest PR!